src/components/Link/link.module.scss

```diff
@@ -1,15 +1,95 @@
 .link-style {
   cursor: pointer;
+  display: inline-block;
   font-family: $octuple-font-family;
   font-size: $text-font-size-2;
+  text-decoration: underline;
+  width: fit-content;
+
+  &.neutral {
```

Comment:
feels like an scss loop waiting to happen

Reply:
This is a SASS sibling selector syntax, scoped to `.link-style`, a loop is unlikely.